test/integration/pkcs-keygen.int.c

```diff
@@ -298,7 +298,7 @@ static void test_destroy(void **state) {
     CK_SESSION_HANDLE session = ti->handle;
 
     CK_BBOOL ck_true = CK_TRUE;
-    CK_BBOOL ck_false = CK_TRUE;
+    CK_BBOOL ck_false = ck_false;
```

Comment:
Looks like this is used for CKA_SENSITIVE, weird that it's causing it to fail....

Reply:
not really weird.
The template sets CKA_SENSITIVE to CK_FALSE, which then sets CKA_ALWAYS_SENSITIVE also to false.
verify_missing_priv_attrs_common in test.c however expects CKA_ALWAYS_SENSITIVE to be true.

Reply:
@williamcroberts  However I'm not sure what is the correct way to fix this.
Change the template to set it to CK_TRUE, (which is not equal to the p11 template?) or change verify_missing_priv_attrs_common (to check the existence of the attribute, but not the value itself, probably needed for CKA_EXTRACTABLE as well)

Reply:
> @williamcroberts However I'm not sure what is the correct way to fix this.
> Change the template to set it to CK_TRUE, (which is not equal to the p11 template?) or change verify_missing_priv_attrs_common (to check the existence of the attribute, but not the value itself, probably needed for CKA_EXTRACTABLE as well)

I want it to match the p11 template, so I would adjust the test.

Reply:
And of course it uncovered a bug #246 :)